src/mcp/prompts/core/index.ts

```diff
@@ -1,3 +1,4 @@
+import { init } from "./init";
 import { deploy } from "./deploy";
 
-export const corePrompts = [deploy];
+export const corePrompts = [deploy, init];
```

Comment:
Stick this behind mcpalpha  experiment flag for the moment (and we'll unflag it later this week)? The guide files are pretty underbaked atm, but I'd love to get this PR in so that we can loop in more folks to work on guides.